### PR TITLE
Prep 1.0.1 release

### PR DESCRIPTION
**Pull request by bdemers**
lumping a few small commits together in one PR, you can review them independently or all at once
The first 2 are pretty isolated, the last three are CI build cleanup

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=false
```
Comment (vijetmahabaleshwar-okta) [inline .editorconfig]: Have we decided that `insert_final_newline` should be false? I thought we needed it. 
Either way, not a big deal.

File: src/ci/release.sh [hunk 2]
```diff
@@ -26,7 +26,7 @@ TAG_NAME="okta-sdk-root-${NEW_VERSION}" # default release plugin tag format
 
 # TODO: we must use our local maven settings file as this script is NOT ready for triggered by travis
```
Comment (vijetmahabaleshwar-okta) [inline src/ci/release.sh]: Is this TODO related to the change you made? i.e. does this still apply?

File: mvnw [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    ls=`ls -ld "$PRG"`
+    link=`expr "$ls" : '.*-> \(.*\)$'`
+    if expr "$link" : '/.*' > /dev/null; then
+      PRG="$link"
+    else
+      PRG="`dirname "$PRG"`/$link"
+    fi
+  done
+
+  saveddir=`pwd`
```
Comment (vijetmahabaleshwar-okta) [inline mvnw]: would `pushd`, `popd` work here?

File: .editorconfig (hunk 1 above)
Comment (bdemers) [reply .editorconfig]: No official resolution yet

File: src/ci/release.sh (hunk 2 above)
Comment (bdemers) [reply src/ci/release.sh]: Still applies, still a little work to do on that front

File: mvnw (hunk 3 above)
Comment (bdemers) [reply mvnw]: all of the `mvnw` is generated from: https://github.com/takari/maven-wrapper
